packages/web/pages/apps/index.tsx

```diff
@@ -189,5 +189,6 @@ export async function getStaticProps() {
     props: {
       apps,
     },
+    revalidate: 86400,
```

Comment:
This will invalidate the next cache once per day, meaning that if a new app is added to github, it will take a maximum of 24 hours to show up. This slightly impacts performance but otherwise we either have to 1) wait for another build (i.e. another merge) or 2) get rid of staticProps (significant impact to performance)